admin-dev/themes/new-theme/js/types/global.d.ts

```diff
@@ -74,33 +74,33 @@ type FetchResponse = Record<string, any>;
 
 type OptionsObject = FetchResponse;
 
-type VTags = Vue & {
+type VTags = {
```

Comment:
Seems like this type is not equivalent to the previous one anymore 🤷‍♂️
Is it because the `Vue` type has disappeared? Maybe it should be replaced by:
```suggestion
type VTags = App & {
```

Reply:
`Vue` doesn't exist anymore as a type, it's now supporting TS properly so we don't need to extend `Vue` anymore